lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -1962,6 +1962,68 @@ void BoundInterpreterFunction::fwdGatherInst(const glow::GatherInst *I) {
   }
 }
 
+template <typename ElemTy>
+void BoundInterpreterFunction::fwdGatherNDInstImpl(
+    const glow::GatherNDInst *I) {
+
+  Tensor *dataT = getTensor(I->getData());
+  auto &dataTy = dataT->getType();
+  Tensor *indicesT = getTensor(I->getIndices());
+  Tensor *outT = getTensor(I->getDest());
+  auto &indicesTy = indicesT->getType();
+
+  // Get the last dimension of indices Tensor
+  const int64_t lastIndicesDimension =
+      indicesTy.dims()[indicesTy.dims().size() - 1];
+
+  size_t out_p = 0;
```

Comment:
nit: 
```suggestion
  size_t outP = 0;
```

Reply:
Done.
@jfix71 Can we land this now?
Thank you